src/renderer/coremods/contextMenu/plaintextPatches.ts

```diff
@@ -7,7 +7,11 @@ export default [
       {
         match: /((\w+)\){)(var\s*\w+;let{navId:)/,
         replace: (_, prefix, menu, suffix) =>
-          `${prefix}replugged.coremods.coremods.contextMenu._insertMenuItems(${menu});${suffix}`,
+          `${prefix}const patchedMenu=replugged.coremods.coremods.contextMenu._buildPatchedMenu(${menu});${suffix}`,
+      },
+      {
+        match: /(patchedMenu.{2400,2600}return)(\(0,.\.jsx\)\(\w+.OnMenuSelectContext)/,
```

Comment:
I can't find this match in sources? Use a find if possible too

Reply:
oh thats because a match works only if the first one replaced. 
i am using find btw?

Reply:
Why would you do it like that? Why not replace it all at once correctly...?

Reply:
because first one runs in start of function and second in the end where it returns

Reply:
solved